migrate/20210623152412_create_tables.go

```diff
@@ -44,8 +44,7 @@ func init() {
 				type       SMALLINT NOT NULL,
 				public_key BYTEA,
 				extra_data JSONB,
-				r          BYTEA,
-				s          BYTEA
+				signature  BYTEA,
```

Comment:
Please create a new migration file using `go run scripts/migrate.go create` to modify this column and add the new table

Reply:
Good point, created!

Reply:
Actually, do you think I should also revert to btcec.Signature in MsgBitCloutTxn (previously changed it to []byte) and r, s in Postgres?